### PR TITLE
Writing_Good_Commits.rst: Add missing blank lines

### PR DESCRIPTION
**Pull request by Cygnus32**
Closes https://github.com/coala/documentation/issues/248

### CODE REVIEW

**Comment by Cygnus32** [comment]
@Adrianzatreanu Done, Please Check :D

File: Developers/Writing_Good_Commits.rst [hunk 1]
```diff
@@ -139,6 +139,7 @@ More Examples
 -------------
```
Comment (Adrianzatreanu) [inline Developers/Writing_Good_Commits.rst]: hey it appears you havent set up your git authorship correctly, you might want to check that before we merge this commit :) its something with git config, google it :)

Comment (AbdealiJK) [inline Developers/Writing_Good_Commits.rst]: @Adrianzatreanu he did set up his git authorship. Just that his email in git config isn't same as the github profile. That's completely fine IMO and some people don't mind the wrong email (I mix up my github and work email all the time)

**Comment by AbdealiJK** [comment]
ack USERINFO_STRUCT

**Comment by AbdealiJK** [comment]
ack 42de07a

**Comment by aptrishu** [comment]
Your commit message lacks information. @Cygnus32 
Go through https://coala.io/commit to know `how to write good commit messages`.

**Comment by sils** [comment]
the commit message is perfectly good IMO, just a rebase and this can be merged!

**Comment by Mixih** [comment]
@Cygnus32 What's your status? Do you need help with the rebase?

**Comment by sils** [comment]
Ok, this will have to happen in coala. @Cygnus32 sorry about this. If you'd like to give feedback on hat we could've done better, please do so at coala.io/newform . We're trying very hard to avoid situations where work is wasted (as we're all volunteers)

**Pull request closed without merging**